src/components/AddBook.js

```diff
@@ -1,13 +1,14 @@
 import React, { useState } from 'react';
 import { v4 as uuidv4 } from 'uuid';
 import { useDispatch } from 'react-redux';
-import { addBook } from '../redux/books/booksSlice';
+import { postBooks, fetchBook } from '../redux/books/booksSlice';
 
 const AddBook = () => {
   const dispatch = useDispatch();
   const [getTitle, setTitle] = useState('');
   const [getAuthor, setAuthor] = useState('');
   const [message, setMessage] = useState('');
+  // const [state, setState] = useState(false);
```

Comment:
- [Optional] Please consider removing any piece of commented code from your code base, commented code makes your project look `unprofessional` and is considered a sign of `code` that is `not clean`.